### PR TITLE
Added Views Files

### PR DESCRIPTION
**Pull request by bbatsche**
- `ad.create.php`
- `ad.index.php`

### CODE REVIEW

**Comment by bbatsche** [comment]
Just adding some boilerplate code for our views

File: public/ad.create.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php  ?>
+<!DOCTYPE html>
```
Comment (ryanorsinger) [inline public/ad.create.php:2]: I'm glad you're doing the HTML5 doctype

File: public/ad.index.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?php  ?>
```
Comment (ryanorsinger) [inline public/ad.index.php:1]: why is this necessary if there isn't any php code yet?

**Comment by ryanorsinger** [comment]
This merge request is good. I'm going to merge.
